examples/pytorch/AxHyperOptimizationPTL/iris_data_module.py

```diff
@@ -28,7 +31,7 @@ def setup(self, stage=None):
 
         # Assign train/val datasets for use in dataloaders
         if stage == "fit" or stage is None:
-            iris_full = self.prepare_data()
+            iris_full = self._get_iris_as_tensor_dataset()
             self.train_set, self.val_set = random_split(iris_full, [130, 20])
```

Comment:
TL;DR
https://github.com/PyTorchLightning/pytorch-lightning/pull/7238 caused the failure. When we call `prepare_data` multiple times, it doesn't return anything after the first call.


```python
self.prepare_data()  # -> returns something
self.prepare_data()  # -> returns nothing
```

Reply:
test passed:

https://github.com/mlflow/mlflow/pull/4348/checks?check_run_id=2565537638

Reply:
https://github.com/mlflow/mlflow/pull/4350 should fix other failures.